plugins/destination_testing.go

```diff
@@ -46,6 +42,13 @@ func getTestLogger(t *testing.T) zerolog.Logger {
 }
 
 func (s *destinationTestSuite) destinationPluginTestWriteOverwrite(ctx context.Context, p *DestinationPlugin, logger zerolog.Logger, spec specs.Destination) error {
+	// These are the indexes for fields in the test-table (without _cq_sync_time and _cq_source_name)
+	// To get the indices in structs that include those fields, add +2
+	const uuidFieldIndex = 5
+	const intFieldIndex = 3
+	const cqSyncTimeFieldIndex = -1
```

Comment:
Is there a way we can infer these indexes from the table instead of hardcoding them?